install_config/persistent_storage/persistent_storage_csi.adoc

```diff
@@ -0,0 +1,358 @@
+[[install-config-persistent-storage-persistent-storage-csi]]
+= Persistent Storage Using Container Storage Interface (CSI)
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+Container Storage Interface (CSI) allows {product-title} to consume storage from storage backends that implement the link:https://github.com/container-storage-interface/spec[CSI interface] as xref:../../architecture/additional_concepts/storage.adoc#architecture-additional-concepts-storage[persistent
+storage].
+
+
+[NOTE]
+====
+CSI volumes is a beta feature and may change in a future release of {product-title}. Due to its beta status, it does not support SELinux yet.
+====
```

Comment:
s/not support SELinux yet./not support SELinux at this time.

Reply:
I removed the whole sentence, we do support SELinux now.